### ISSUE TITLE
html-webpack-plugin-after-emit shows deprecated note

### ISSUE THREAD

**Issue by jantimon** [ampedandwired/html-webpack-plugin#467]
The html-webpack-plugin-after-emit event shows a deprecated note although you cant return anything.

@ascariandrea do you have time to look into this?

**Comment by ascariandrea** [comment]
Sure! I'll do it in a few days!

**Comment by ascariandrea** [comment]
Can you assign me this issue?

**Comment by jantimon** [comment]
don't got the permissions sry ;)

**Comment by ascariandrea** [comment]
I don't understand where is the problem:
- What should be the right behaviour?

I tested and the warning is rightly displayed when no result is given to the `callback()`.

**Comment by jantimon** [comment]
Yes that's perfect - but for emit there is nothing to return

**Comment by octref** [comment]
Related: https://github.com/ampedandwired/html-webpack-plugin/pull/442

This is what vue's webpack template is using.

```
    new HtmlWebpackPlugin({
      filename: 'index.html',
      template: 'index.html',
      inject: true
    })
```

I don't know what this means `"Using html-webpack-plugin-after-html-processing without returning a result is deprecated."`. From my POV I'm only passing 3 static config and there is nothing to return.

Please fix this as this error doesn't tell me what's wrong or how to fix it.

**Comment by okcjosh** [comment]
Correct. It is impossible to fix actually depending upon your build process. I am in the same boat. There is literally nothing to be returned. Need this fixed or need a workaround ASAP.

**Comment by jantimon** [comment]
@okcjosh could you please debug this and tell us which plugin is causing this message?

**Comment by okcjosh** [comment]
Html-webpack-plugin. Heres how I’m using it. 

let htmlConfig = {
    template: 'client/_index.html',
    filename: '../client/index.html',
    alwaysWriteToDisk: true
};
config.plugins.push(
  new HtmlWebpackPlugin(htmlConfig),
  new HtmlWebpackHarddiskPlugin()
);

**Comment by jantimon** [comment]
Resolved by #478 thanks @ascariandrea
